xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -854,6 +857,7 @@ unsigned int CAESinkAUDIOTRACK::AddPackets(uint8_t **data, unsigned int frames,
       }
 
       usleep(extra_sleep * 1000);
+      CLog::Log(LOGINFO, "m_delay %lf ms Stream Duration %lf Adding-Time %lf Sleep-Time %lf ms", m_delay * 1000, m_format.m_streamInfo.GetDuration(), time_to_add_ms, extra_sleep);
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)